src/libraries/Pool.sol

```diff
@@ -248,7 +248,7 @@ library Pool {
         }
 
         // Fees earned from LPing are removed from the pool balance.
```

Comment:
Maybe we could make this comment more clear now... 
// Fees earned from LPing are added to the user's collectable amount.